CoffeeShop/src/main/java/order/OrderList.java

```diff
@@ -67,6 +74,22 @@ public Boolean remove(UUID ID) {
         return false;
     }
 
+    /**
+     * Method to remove and return the first order in the queue
+     *
+     * @return Order object to be processed by staff
+     */
+    public Order remove() {
+        if (inCompleteOrders.isEmpty()) {
+            return null;
+        }
+        return inCompleteOrders.poll();
+    }
```

Comment:
```suggestion
    public Order remove() {
        return inCompleteOrders.poll();
    }
```

we can simplify this as .poll will [return null](https://docs.oracle.com/javase/7/docs/api/java/util/Queue.html) if inCompleteOrders is empty